### ISSUE TITLE
docs: update participant contributions

### ISSUE THREAD

**Issue by FitzsimmonsCM** [iRNA-COSI/APAeval#111]
Updating the README with participant contributions using all-contributors bot

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @ninsch3000 for code,data,doc,design,eventOrganizing,mentoring,projectManagement,question,review,talk

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @ninsch3000 for code, data, doc, design, eventOrganizing, mentoring, projectManagement, question, review, talk

**Comment by chelseaherdman** [comment]
approved it...so did you figure out the branching issue? How can I help?

**Comment by FitzsimmonsCM** [comment]
Yeah, it was my mistake. We did the initial test as a comment on issue #71 but I tagged the `bot` on a few other issues before that test was merged. As a result, the bot thought it needed to create a rc file for each new request I made--because it didn't find one in `main` But it should be ok moving forward.

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @EuancRNA for code, mentoring, question

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @chelseaherdman for eventOrganizing, ideas, projectManagement, review, talk

**Comment by FitzsimmonsCM** [comment]
Ok, third times the charm @EuancRNA 

@all-contributors please add @EuancRNA for code, ideas, mentoring

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @uniqueg for bug, code, doc, example, eventOrganizing, ideas, infra, maintenance, mentoring, projectManagement, question, review, talk

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @yuukiiwa for bug, blog, code, data, doc, example, eventOrganizing, ideas, mentoring, projectManagement, question, test, tutorial

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @BenNicolet for data, eventOrganization, ideas, projectManagement

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @pjewell-biociphers maintenance

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @mzavolan for data, doc, eventOrganizing, financial, findingFunding, ideas, mentoring, projectManagement, question, research, review, talk

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @mfansler for bug, code, doc, eventOrganizing, ideas, mentoring, projectManagement, question, review

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @mkatsanto for code, ideas, mentoring, question

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @daneckaw for code, data, eventOrganizing, ideas, projectManagement, tutorial

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @dominikburri for bug, code, data, doc, example, eventOrganizing, ideas, infra, mentoring, projectManagement, question

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @mrgazzara for code, doc, data, eventOrganizing, ideas, infra, maintenance, projectManagement, mentoring, talk

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @FitzsimmonsCM for docs, eventOrganizing, ideas, projectManagement, talk

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @lschaerfen for code, ideas

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @lschaerfen for code, ideas, talk

**Comment by chelseaherdman** [comment]
@all-contributors please add @chilampoon for code, ideas, question

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @dseyres for code, docs, ideas

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @plger for data, doc, eventOrganizing, ideas, projectManagement

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @plger for data, doc, eventOrganizing, ideas, projectManagement

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @plger for data, doc, eventOrganizing, ideas, projectManagement

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @SamBryce-Smith for code, ideas

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @dominikburri for test, tutorial

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @pinjouwu9325 for code, ideas

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @yoseopyoon for code, ideas

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @SamBryce-Smith for bug, code, doc, ideas, maintenance, mentoring, projectManagement, question, review, tutorial, talk

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @faricazjj for bug, code, doc, ideas, maintenance, projectManagement, question, review

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @txellferret for code, example, ideas, infra, mentoring, question

**Comment by FitzsimmonsCM** [comment]
@all-contributors please add @grexor for bug, code, ideas, maintenance, review

**Comment by ninsch3000** [comment]
@all-contributors please add @jmfernandez for ideas, infra, mentoring
